src/Aspire.ProjectTemplates/templates/aspire-empty/.template.config/template.json

```diff
@@ -21,7 +21,8 @@
     "type": "solution",
     "editorTreatAs": "solution",
     "aspire-8.2-tfms": "net8.0",
-    "aspire-9.0-tfms": "net8.0;net9.0"
+    "aspire-9.0-tfms": "net8.0;net9.0",
```

Comment:
Just out of curiosity (not super familiar with these files) how come these tags aren't in all template configs? For instance, aspire-apphost doesn't seem to have this change. Is that intentional?

Reply:
they're only used by the code in VS that drives the solution explorer "Add Aspire orchestration" context menu.